### PR TITLE
Create routes dynamically

### PR DESCRIPTION
**Pull request by dkryvi**
#2

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (Vanish404) [inline .vscode/launch.json]: do we need this file?

File: lib/config/defaults.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "server": {
-    "port": 3000
+    "port": 3000,
+    "routesFolder": "lib/routes"
```
Comment (akosenko) [inline lib/config/defaults.json:4]: lib - this name confused me. But if we are ok with it - I'll skip this moment in future

File: .vscode/launch.json (hunk 1 above)
Comment (dkryvi) [reply .vscode/launch.json]: It's file help to create debug config for vs-code automatically(like webstorm config to start project)

Comment (akosenko) [reply .vscode/launch.json]: so, we should delete this kind of file

Comment (akosenko) [reply .vscode/launch.json]: and add this folder to .gitignore

File: lib/utils/initialize_routes.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  const initialRouter = express.Router();
+
+  if (isNil(folder)) {
+    return initialRouter;
+  }
+
+  const cwd = process.cwd();
+
+  return glob.sync('*.js', { cwd: folder }).reduce((router, fileName) => {
+    const routeName = get(fileName.split('.'), '0');
```
Comment (akosenko) [inline lib/utils/initialize_routes.js:15]: why we should do this trick?

File: .vscode/launch.json (hunk 1 above)
Comment (dkryvi) [reply .vscode/launch.json]: Where should I write this config? I use this config for debugging server in vs-code

File: lib/config/defaults.json (hunk 2 above)
Comment (dkryvi) [reply lib/config/defaults.json:4]: What name of the folder should we use? if we will use `app` name we have import like `./app/services/app` when import app service.

File: .vscode/launch.json (hunk 1 above)
Comment (akosenko) [reply .vscode/launch.json]: ah... are you using node_modules? Why you don't push them and such folder as .idea/*?
Also you already have such line in .gitignore file but with mistake :)
./vscode/* should be replaced with ./.vscode/*

File: lib/config/defaults.json (hunk 2 above)
Comment (akosenko) [reply lib/config/defaults.json:4]: nevermind... skip this - I'll refactor this code

File: .vscode/launch.json (hunk 1 above)
Comment (akosenko) [reply .vscode/launch.json]: @DenKrivichanin please let me know if you have any questions

**Review by akosenko — APPROVED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (dkryvi) [reply .vscode/launch.json]: I'll fix .gitignore file. But why it's mistake to share config for debugging node js server in some editor?

**Pull request closed without merging**